R/cronbach.R

```diff
@@ -0,0 +1,39 @@
+#' @title Cronbach's Alpha for Items or Scales
+#' @name cronbach
+#'
+#' @description Compute various measures of internal consistencies
+#'    for tests or item-scales of questionnaires.
+#'
+#' @param x A matrix or a data frame.
+#'
+#' @return The Cronbach's Alpha value for \code{x}.
+#'
+#' @details The Cronbach's Alpha value for \code{x}. A value closer to 1
+#'    indicates greater internal consistency, where usually following
+#'    rule of thumb is applied to interprete the results:
+#'    \ifelse{html}{\out{&alpha;}}{\eqn{\alpha}{alpha}} < 0.5 is unacceptable,
+#'    0.5 < \ifelse{html}{\out{&alpha;}}{\eqn{\alpha}{alpha}} < 0.6 is poor,
+#'    0.6 < \ifelse{html}{\out{&alpha;}}{\eqn{\alpha}{alpha}} < 0.7 is questionable,
+#'    0.7 < \ifelse{html}{\out{&alpha;}}{\eqn{\alpha}{alpha}} < 0.8 is acceptable,
+#'    and everything > 0.8 is good or excellent.
+#'
+#' @examples
+#' data(mtcars)
+#' x <- mtcars[, c("cyl", "gear", "carb", "hp")]
+#' cronbach(x)
+#'
+#' @importFrom stats var na.omit
+#' @export
+cronbach <- function(x) {
```

Comment:
Would `cronbach_alpha()` be more transparent (or is Cronbach's only legacy this metric ^^?)

Reply:
> or is Cronbach's only legacy this metric

I don't know, you are the psychologist! :-)
In times of auto-completion, I would say we can use `cronbachs_alpha()`.

Reply:
It can also increase its google's findability 👀    (sometimes I wonder if my psychologist-part is getting slowly taken over by a marketing/salesman part 😱 )